### PR TITLE
Add article on how to contribute samples

### PR DESCRIPTION
**Pull request by cgranade**
As we continue standardizing sample layouts and onboarding Quantum Development Kit samples to the [Samples Browser](docs.microsoft.com/samples), it will be helpful to document how we write samples and what content needs to be there for working with the Samples Browser. This PR begins that effort by documenting what I look for as I port samples to that format and prepare them for publication at docs.microsoft.com/samples.

### CODE REVIEW

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards

**A minimum Acrolinx score of 80 is required.**

Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology.

| Article | Acrolinx<br>score | Lowest<br>category<br>score | Spelling<br>issues | Scorecard | Processed |
|---------|:--------------:|:--------------------:|:------:|:---------:|:---------:|
| [articles/contributing/code.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/6470ec01d7add6391ba32f58888c3ea0fdd8b2b8/articles/contributing/code.md) | 78 | 57 | 0 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/d3d43b59-b37a-46ea-b67a-664ee73d609b_report.html) | [:white_check_mark:](# "Successfully processed") |
| [articles/contributing/docs.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/6470ec01d7add6391ba32f58888c3ea0fdd8b2b8/articles/contributing/docs.md) | 83 | 53 | 2 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/f43abe4b-399b-4f04-817f-aa4d96d0454f_report.html) | [:white_check_mark:](# "Successfully processed") |
| [articles/contributing/index.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/6470ec01d7add6391ba32f58888c3ea0fdd8b2b8/articles/contributing/index.md) | 87 | 67 | 0 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/768143dc-a565-473f-9e81-020beafc2a20_report.html) | [:white_check_mark:](# "Successfully processed") |
| [articles/contributing/samples.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/6470ec01d7add6391ba32f58888c3ea0fdd8b2b8/articles/contributing/samples.md) | 85 | 67 | 1 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/be05e45f-e9c7-439b-be38-42aefb8993c9_report.html) | [:white_check_mark:](# "Successfully processed") |
| [articles/contributing/toc.yml](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/6470ec01d7add6391ba32f58888c3ea0fdd8b2b8/articles/contributing/toc.yml) | 100 | 100 | 0 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/0a4bff5c-c076-4f30-a9ff-99a83143b118_report.html) | [:white_check_mark:](# "Successfully processed") |

**More information about Acrolinx**

- [Install Acrolinx locally for VSCode](https://review.docs.microsoft.com/help/contribute/contribute-acrolinx-vscode)
- [False positives or issues](https://aka.ms/acrolinxbug)
- [Troubleshooting issues with Acrolinx](https://review.docs.microsoft.com/help/contribute/acrolinx-error-messages)

**Comment by KittyYeungQ** [comment]
Thank you so much @cgranade I really appreciate it. I've reviewed and approved but would like @efratshabtai to take a look at it as well, before merging.

File: articles/contributing/samples.md [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+  📁 chemistry/
+   ⋮
+```
+
+That is, the samples in the [microsoft/Quantum repository](https://github.com/microsoft/Quantum) are broken down by subject area into different folders such as `algorithms/`, `arithmetic/`, or `characterization/`.
+Within the folder for each subject area, each sample consists of a single folder that collects everything a user will need to explore and make use of that sample.
+
+## How Samples are Structured
+
+Looking at the files that make up each folder, let's dive into the [`algorithms/chsh-game/`](todo:todo) sample.
```
Comment (geduardo) [inline articles/contributing/samples.md]: Don't forget to add the correct link here.

Comment (cgranade) [reply articles/contributing/samples.md]: Ah, thanks for catching that. Didn't have it right at hand and forgot to add it back.

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards

**A minimum Acrolinx score of 80 is required.**

Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology.

| Article | Acrolinx<br>score | Lowest<br>category<br>score | Spelling<br>issues | Scorecard | Processed |
|---------|:--------------:|:--------------------:|:------:|:---------:|:---------:|
| [articles/contributing/code.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/5ade66224d5c34bbf34f289ab6d6d06c382a670c/articles/contributing/code.md) | 78 | 57 | 0 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/e60a89b4-b4bb-4cad-b1e6-cdefac2775da_report.html) | [:white_check_mark:](# "Successfully processed") |
| [articles/contributing/docs.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/5ade66224d5c34bbf34f289ab6d6d06c382a670c/articles/contributing/docs.md) | 83 | 53 | 2 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/163372b3-6c60-41fa-943f-b2893094ef6c_report.html) | [:white_check_mark:](# "Successfully processed") |
| [articles/contributing/index.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/5ade66224d5c34bbf34f289ab6d6d06c382a670c/articles/contributing/index.md) | 87 | 67 | 0 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/c430948d-0e15-44fe-9564-1daec7932ebe_report.html) | [:white_check_mark:](# "Successfully processed") |
| [articles/contributing/samples.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/5ade66224d5c34bbf34f289ab6d6d06c382a670c/articles/contributing/samples.md) | 85 | 67 | 1 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/1e4224be-cdc1-49ab-a40e-7e51a901e613_report.html) | [:white_check_mark:](# "Successfully processed") |
| [articles/contributing/toc.yml](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/5ade66224d5c34bbf34f289ab6d6d06c382a670c/articles/contributing/toc.yml) | 100 | 100 | 0 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/8b1e6970-fcc6-4f5d-a228-1c94a42004ad_report.html) | [:white_check_mark:](# "Successfully processed") |

**More information about Acrolinx**

- [Install Acrolinx locally for VSCode](https://review.docs.microsoft.com/help/contribute/contribute-acrolinx-vscode)
- [False positives or issues](https://aka.ms/acrolinxbug)
- [Troubleshooting issues with Acrolinx](https://review.docs.microsoft.com/help/contribute/acrolinx-error-messages)

File: articles/contributing/samples.md [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+## Anatomy of a Helpful README File
+
+One especially important file, though, is the `README.md` file, as that's what users need to get started with your sample!
+
+Each `README.md` should start with some metadata that helps docs.microsoft.com/samples find your contribution.
+
+> [!div class="nextstepaction"]
+> [See how the chsh-game sample is rendered](https://docs.microsoft.com/samples/microsoft/quantum/validating-quantum-mechanics/)
+
+This metadata is provided as a [YAML header](https://dotnet.github.io/docfx/spec/docfx_flavored_markdown.html#yaml-header) that indicates what languages your sample covers (typically, this will be `qsharp`, `csharp`, and `python`), and what products your sample covers (typically, just `qdk`).
```
Comment (efratshabtai) [inline articles/contributing/samples.md]: Is this mandatory (tagging a language and a product)?
If so then we should say it explicitly as it seems that the rest of the ReadMe instructions are recommendations only.

**Pull request merged**